android/src/main/java/com/swmansion/reanimated/layoutReanimation/Snapshot.java

```diff
@@ -62,25 +62,40 @@ public class Snapshot {
     view.getLocationOnScreen(location);
     globalOriginX = location[0];
     globalOriginY = location[1];
+    initKeys();
+  }
+
+  public Snapshot(View view) {
+    int[] location = new int[2];
+    view.getLocationOnScreen(location);
+    originX = location[0];
+    originY = location[1];
+    width = view.getWidth();
+    height = view.getHeight();
+    initKeys();
```

Comment:
what's the point of initializing static fields in instance constructor. It seems like this should be initialized statically (e.g. use `static` block in this class or just assign in-place with the field definition.